app/javascript/packs/localizeArticleDates.js

```diff
@@ -0,0 +1,12 @@
+/* Show article date/time according to user's locale */
+import { addLocalizedDateTimeToElementsTitles } from '../utilities/localDateTime';
+
+const initializeArticleDate = () => {
+  const articlesDates = document.querySelectorAll(
+    '.crayons-story time, article time',
```

Comment:
Removed the `.single-other-article time` selector because we no longer use that class: see #10416